src/router/index.js

```diff
@@ -15,6 +16,11 @@ const routes = [
     path: '/roles',
     name: 'Roles',
     component: Roles
+  },
+  {
```

Comment:
why are you creating a "Login" route when you unload the router when the user is logged off and just show the Login view?
if you are going to use a login route, you should do a redirect in App.vue instead of unloading the router.

Reply:
I was just for testing purposes. Removed and solved.